### ISSUE TITLE
teleporting to a friend doesnt work

### ISSUE THREAD

**Issue by mgracer48** [toontownfellowship/src#113]
It sends you back to playground , workaround is teleport to estate or another district or another zone then teleport to the friend

**Comment by pslifeisgood** [comment]
https://github.com/toontownfellowship/src/issues/111

**Comment by Loudrob** [comment]
This issue isn't really descriptive enough. Where is your friend when this happens? Street, their estate, cog hq?

**Comment by mgracer48** [comment]
it is when you teleport to someone on the street you get " The toon was moving around please try again." Or something like that

**Comment by Loudrob** [comment]
Alright, this issue is already known of I believe.

**Comment by gamesawdrew** [comment]
Yes this issue is very annoying... It happens even if they are standing still..

**Comment by enigmeow** [comment]
from the last time I dug through this: https://github.com/ToontownInfinite/src/issues/206

It has bubbled up to the top of my "most annoying" bugs so I'll dig into it again since nobody else has managed to kill it yet.. :)

**Comment by Xenofied** [comment]
This has been fixed in 1.1.3

**Comment by mgracer48** [comment]
really?
